src/events.js

```diff
@@ -2826,23 +1716,25 @@ export const addEventShipPrintManifest = (id) => {
   })
 }
 
-export const addEventNavBarBoxManifest = (id, userName) => {
+export const addEventNavBarBoxManifest = (id) => {
+    const userName = appState.getState().userName;
     const btn = document.getElementById(id);
     document.getElementById(id).addEventListener('click', e => {
         e.stopPropagation();
         if (btn.classList.contains('active')) return;
         if (id == 'viewBoxManifestBlood') {
```

Comment:
Not related this PR but is should be `===`

Reply:
Agree & updated on 1725 an 1729.